### PR TITLE
Fix linux installation instructions

### PR DESCRIPTION
**Pull request by floriangosse**
### Proposed changes

Updated the installation instructions for Debian and Ubuntu. Some of the steps in the instruction were wrong and resulted in an error.

### CODE REVIEW

File: engine/installation/linux/ubuntu.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ Docker from the repository.
 
     ```bash
     $ sudo add-apt-repository \
-           "deb https://apt.dockerproject.org/repo/pool/ \
+           "deb https://apt.dockerproject.org/repo \
            $(lsb_release -cs) \
```
Comment (thaJeztah) [inline engine/installation/linux/ubuntu.md]: this should be `ubuntu-$(lsb_release -cs)`

File: engine/installation/linux/ubuntu.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ Docker from the repository.
 2.  Add Docker's official GPG key:
 
     ```bash
-    $ curl -s http://yum.dockerproject.org/gpg | sudo apt-key add
+    $ curl -s http://yum.dockerproject.org/gpg | sudo apt-key add -i
```
Comment (thaJeztah) [inline engine/installation/linux/ubuntu.md]: I think this should be `-` as well, not `-i`?

Comment (floriangosse) [inline engine/installation/linux/ubuntu.md]: Oh, yes it is a typo. I will fix it.

File: engine/installation/linux/ubuntu.md (hunk 1 above)
Comment (floriangosse) [inline engine/installation/linux/ubuntu.md]: I will fix it.

**Comment by jsangilve** [comment]
this resolves issues #1218 and #1249

**Comment by mstanleyjones** [comment]
Yeah these changes were duplicated by #1224. Thank you!

**Comment by floriangosse** [comment]
This PR doesn't fix all the bugs in the documentation. It only fixes the key import.

**Comment by mstanleyjones** [comment]
You're right, but they have all been duplicated elsewhere. I rebased your commit and there are now 0 files changed by your commit. Please let me know if I'm missing anything.

**Comment by floriangosse** [comment]
You're right I didn't realized that a change for the distro prefix was merged.

**Pull request closed without merging**